### ISSUE TITLE
Dealing with overfitting

### ISSUE THREAD

**Issue by igorbpf** [yoonkim/CNN_sentence#34]
Does anyone know how overfitting was dealt? I read something about early stopping at 3.1 section, but I get overfitting  at second epoch of training using the hyperparameters specified in the article. Is that correct?

**Comment by hkhatod** [comment]
regularization: l2 or dropout or both.
